pkgs/by-name/ld/ld64/package.nix

```diff
@@ -182,17 +154,24 @@ stdenv.mkDerivation (finalAttrs: {
 
     cd "$NIX_BUILD_TOP/$sourceRoot"
 
-    export NIX_CFLAGS_COMPILE+=" --ld-path=$out/bin/${targetPrefix}ld"
+    export NIX_CFLAGS_COMPILE+=" --ld-path=$out/bin/ld"
     export NIX_CFLAGS_LINK+=" -L$SDKROOT/usr/lib"
-    meson setup build-install-check -Db_lto=true --buildtype=$mesonBuildType${
-      lib.optionalString (targetPrefix != "") " -Dtarget_prefix=${targetPrefix}"
+    meson setup build-install-check --buildtype=$mesonBuildType ${
+      lib.escapeShellArgs [
+        (lib.mesonBool "b_lto" true)
+        (lib.mesonOption "libllvm_path" "${lib.getLib llvm}")
+      ]
```

Comment:
Same nit, but also is there anything we can do to avoid the duplication here?

Reply:
I fixed the nits and refactored the `lib.getLib llvm` to a variable.